From 830f900cd99088eb90501d02d5ad31dd5b8e810f Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Tue, 22 Jun 2021 09:46:59 +0000
Subject: libstdc++: remove format include

format is C++20 feature that is not available in libstdc++ yet. But
it is also not required in this context.

Bug: 957519
Change-Id: Ic17d9815bd8aca7e597c30cace9a66cee1ea63b9
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2972980
Reviewed-by: Tommy Nyquist <nyquist@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/master@{#894598}
---
 .../internal/execution/model_execution_manager_impl.cc           | 1 -
 1 file changed, 1 deletion(-)

diff --git a/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc b/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc
index 4c6e896f875a..bd78bed6e893 100644
--- a/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc
+++ b/components/segmentation_platform/internal/execution/model_execution_manager_impl.cc
@@ -5,7 +5,6 @@
 #include "components/segmentation_platform/internal/execution/model_execution_manager_impl.h"
 
 #include <deque>
-#include <format>
 #include <map>
 #include <memory>
 #include <vector>
-- 
2.30.2

